eZ/Publish/Core/Repository/LocationService.php

```diff
@@ -428,6 +429,11 @@ public function createLocation(ContentInfo $contentInfo, LocationCreateStruct $l
             $this->persistenceHandler->locationHandler()->load($locationCreateStruct->parentLocationId)
         );
 
+        $locationCreateStruct->sortField = $locationCreateStruct->sortField
+            ?? ($content->getContentType()->defaultSortField ?? Location::SORT_FIELD_NAME);
+        $locationCreateStruct->sortOrder = $locationCreateStruct->sortOrder
+            ?? ($content->getContentType()->defaultSortOrder ?? Location::SORT_ORDER_ASC);
```

Comment:
Please extract `$content->getContentType()` to a variable, given it's used two times.

Reply:
Fixed in: https://github.com/ezsystems/ezpublish-kernel/pull/2977/commits/2a12c7298508a1d1a5152c199ffd2944c7ad4cb9